src/NetAppFiles/NetAppFiles/ChangeLog.md

```diff
@@ -19,6 +19,49 @@
 -->
 ## Upcoming Release
 
+## Version 0.1.7
```

Comment:
Please do not add or update any versions in changelog because they are calculated according to sementic versioning at release time. Simply remove this line

Reply:
Actually let me remove this for you as it seems to be the only issue blocking merging

Reply:
Thank you